### PR TITLE
feat: add identity wallet documentation.

### PR DESCRIPTION
**Pull request by Nebulis**

### CODE REVIEW

File: docs/component/identity-wallet.md [hunk 1]
```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
...
+## Installation
+
+For the moment, the application is not available on Apple store or Android store. You will need to run it through [Expo](https://expo.io/).
+
+1. Download Expo [![Expo on apple store](/docs/component/identity-wallet/apple-store.svg)](https://apps.apple.com/fr/app/expo-client/id982107779 "Open expo on apple store") [![Expo on apple store](/docs/component/identity-wallet/android-store.svg "Open expo on android store")](https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en)
+1. Open Expo and download Identity Wallet app by scanning the following QR Code
+
+<img src="/docs/component/identity-wallet/identity-wallet-qr-code.png" alt="identity wallet qr code" title="scan the qr code with expo" width="200"/>
+
+<img src="/img/common/party-popper.png" alt="Congratulations" style="display:inline-block; width: 1.3rem"/> Congratulations, you are now able to use Identity Wallet application.
```
Comment (yehjxraymond) [inline docs/component/identity-wallet.md]: Wanna just use emoji here instead? 🎉

Comment (Nebulis) [reply docs/component/identity-wallet.md]: done =)

File: docs/component/identity-wallet.md [hunk 2]
```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
...
 
-Overview
+OA provides it's own mobile application to view, store and share Verifiable Document and Transferable Records.
 
-## Header
+## Installation
+
+For the moment, the application is not available on Apple store or Android store. You will need to run it through [Expo](https://expo.io/).
+
+1. Download Expo [![Expo on apple store](/docs/component/identity-wallet/apple-store.svg)](https://apps.apple.com/fr/app/expo-client/id982107779 "Open expo on apple store") [![Expo on apple store](/docs/component/identity-wallet/android-store.svg "Open expo on android store")](https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en)
```
Comment (simboonlong) [inline docs/component/identity-wallet.md]: maybe use US link instead? [https://apps.apple.com/us/app/expo-client/id982107779]

Comment (sumitnitsurat) [inline docs/component/identity-wallet.md]: Download Expo Client app.

File: docs/component/identity-wallet.md [hunk 3]
```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
...
+1. Open Expo and download Identity Wallet app by scanning the following QR Code
+
+<img src="/docs/component/identity-wallet/identity-wallet-qr-code.png" alt="identity wallet qr code" title="scan the qr code with expo" width="200"/>
```
Comment (sumitnitsurat) [inline docs/component/identity-wallet.md]: confuse from here. Can not see any option to scan the QR code in Expo Client app.

File: docs/component/identity-wallet.md [hunk 4]
```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
...
+
+🎉 Congratulations, you are now able to use Identity Wallet application.
+
+## Documents
+
+In order to view a document, open Identity Wallet, proceed to the QR Code tab and scan. Some examples are available on [Open Attestation Gallery](http://gallery.openattestation.com/).
+
+In the gallery you will find 2 different kinds of document, viewable and storable:
+- Viewable documents are documents that you can only view in the application. No other action is permitted.
```
Comment (sumitnitsurat) [inline docs/component/identity-wallet.md]: don't need documents again. just "Viewable documents are that you can only view in the application"

File: docs/component/identity-wallet.md (hunk 2 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: why US ? this one is without any region :thinking:

File: docs/component/identity-wallet.md (hunk 3 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: Hmmm if I open the app the first thing displayed is "scan qr code" ?

Comment (sumitnitsurat) [reply docs/component/identity-wallet.md]: I already had expo client app in my mobile. but can not see the option to scan any where. only  the screen shown in the app store and its weird dont even see the plus icon in my app on the top.

Comment (Nebulis) [reply docs/component/identity-wallet.md]: Iphone / Android ?

Print screen of what you have ?

File: docs/component/identity-wallet.md (hunk 2 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: done

File: docs/component/identity-wallet.md (hunk 4 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: Is there a missing word ? Doesn't sound valid to me.

I think repetition is fine in this case

File: docs/component/identity-wallet.md (hunk 2 above)
Comment (simboonlong) [reply docs/component/identity-wallet.md]: oh, i thought that the app page itself in english might be understandable, currently is in french.

Screenshot 2020-02-21 at 9 35 13 AM

File: docs/component/identity-wallet.md (hunk 4 above)
Comment (sebastianquek) [reply docs/component/identity-wallet.md]: In the gallery you will find 2 different kinds of documents, viewable and storable:
- Viewable documents can only be viewed (and not stored) in the application. No other actions are permitted.
- Storable documents can be viewed and stored in the application. Once a document is stored into your wallet you can also share it with others. When you view the document, a QR Code icon will be displayed. Click on it to share your document.

File: docs/component/identity-wallet.md [hunk 5]
```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
 sidebar_label: Identity Mobile App
 ---
 
-Overview
+OA provides it's own mobile application to view, store and share Verifiable Document and Transferable Records.
```
Comment (sebastianquek) [inline docs/component/identity-wallet.md]:
```suggestion
OA provides its own mobile application to view, store and share Verifiable Documents and Transferable Records.
```

File: docs/component/identity-wallet.md (hunk 2 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: how come it's in french .....................

Comment (Nebulis) [reply docs/component/identity-wallet.md]: oh yeah there is the fr I didnt notice thanks

Comment (Nebulis) [reply docs/component/identity-wallet.md]: Changed to https://apps.apple.com/app/expo-client/id982107779 => no lang

File: docs/component/identity-wallet.md (hunk 3 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: ok will keep like this for the moment, until Raymond release on app store =)

File: docs/component/identity-wallet.md (hunk 5 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: done

File: docs/component/identity-wallet.md (hunk 4 above)
Comment (Nebulis) [reply docs/component/identity-wallet.md]: changed

File: docs/component/identity-wallet.md [hunk 6]
```diff
@@ -4,8 +4,28 @@ title: Identity Mobile App
...
+
+In order to view a document, open Identity Wallet, proceed to the QR Code tab and scan. Some examples are available on [Open Attestation Gallery](http://gallery.openattestation.com/).
+
+In the gallery you will find 2 different kinds of documents, viewable and storable:
+- Viewable documents can only be viewed (and not stored) in the application. No other actions are permitted.
+- Storable documents can be viewed and stored in the application. Once a document is stored into your wallet you can also share it with others. When you view the document, a QR Code icon will be displayed. Click on it to share your document.
+
+## Additional information
+- Identity Wallet implementation follow our [Universal Actions ADR](https://github.com/Open-Attestation/adr/blob/master/universal_actions.md).
+- Found a bug ? Having a question ? Want to share an idea ? Reach us out on the [Github repository](https://github.com/Open-Attestation/identity-wallet).
```
Comment (Nebulis) [inline docs/component/identity-wallet.md]:
```suggestion
- Found a bug? Have a question? Want to share an idea? Reach us on the [Github repository](https://github.com/Open-Attestation/identity-wallet).
```

**Pull request merged**